R/RcppExports.R

```diff
@@ -0,0 +1,276 @@
+# Generated by using Rcpp::compileAttributes() -> do not edit by hand
```

Comment:
if this is a generated file, can we ... not have it in the source?

Reply:
I would argue that it needs to be included. Then use wrapper function around these automatic generated functions to add the custom R code (which is now included in RccpExportsManual.R) and divide over several R scripts with some logical division of functionality. This is then just for clarity of the code.